docs/includes/mac-cache-folders.md

```diff
@@ -1,6 +1,6 @@
 Add-ins are often cached in Office on Mac for performance reasons. Normally, the cache is cleared by reloading the add-in. If more than one add-in exists in the same document, the process of automatically clearing the cache on reload might not be reliable.
 
```

Comment:
Consider adding H3 headings for clearing the cache via the personality menu and for manually deleting host folder contents.